src/components/general/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from "react";
+import * as PropTypes from "prop-types";
+import * as styles from "./styles.less";
+
+type TabsProps = {
+    onTabClick: (tabKey: string) => void,
+    activeTabKey: string,
+    children: any,
+};
+
+const Tabs = ({ onTabClick, activeTabKey, children }: TabsProps) => {
+    const renderTabContent = () => {
+        const active = children.find(child => child.props.tabKey === activeTabKey);
+        if (!active) {
+            return null;
+        }
+
+        return React.Children.map(active.props.children, child => React.cloneElement(child));
+    };
+    const renderTabPane = () => {
```

Comment:
Same thing as renderTabContent. Extract this to a seperate component and then: `<TabPane onChange={onChange} activeTabKey={activeTabKey}>{children</TabPane>`